gis/mod_tile/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           active_variants 1.1
 PortGroup           boost 1.0
 PortGroup           github 1.0
 
-github.setup        openstreetmap mod_tile 0.7.2 v
+github.setup        openstreetmap mod_tile 0.8.1 v
 github.tarball_from archive
 revision            1
```

Comment:
the `revision` field should be reset to `0` when updating the `version`

Reply:
Fixed, thanks!